src/super_gradients/training/transforms/all_transforms.py

```diff
@@ -118,3 +120,6 @@
     Transforms.RandomAutocontrast: RandomAutocontrast,
     Transforms.RandomEqualize: RandomEqualize,
 }
+
+ALBUMENTATIONS_TRANSFORMS = {name: cls for name, cls in inspect.getmembers(importlib.import_module("albumentations"), inspect.isclass)}
```

Comment:
inspect.isclass & issubclass(A.BasicTransform) ?

Reply:
Good point, added in https://github.com/Deci-AI/super-gradients/commit/7ec67fb008f8880c4b75348f10d865cfa5ede9b1